### DISCUSSION TITLE
forms/chromium/convert/url and pagenumbers

### DISCUSSION THREAD

**Discussion by Peetzer** [gotenberg/gotenberg#724] [Q&A] [answered]
Is it possible to add (optional) page numbers ( header/ footer) to a pdf from url?

Kind regards, Peter

**Comment by gulien** [comment, marked as answer]
Yes, please read the documentation: https://gotenberg.dev/docs/routes#header--footer

**Reply by Peetzer** [reply]
Thx @gulien!
